AutomatedTellerMachine/src/main/java/ru/sber/projectATM/AutomatedTellerMachine/controller/controllerATM.java

```diff
@@ -0,0 +1,100 @@
+package ru.sber.projectATM.AutomatedTellerMachine.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import ru.sber.projectATM.AutomatedTellerMachine.dto.BalanceDTO;
+import ru.sber.projectATM.AutomatedTellerMachine.exception.ATMInternalErrorException;
+import ru.sber.projectATM.AutomatedTellerMachine.service.ATMService;
+import ru.sber.projectATM.AutomatedTellerMachine.validate.*;
+import ru.sber.projectATM.common.*;
+
+
+import ru.sber.projectATM.common.handbook.*;
+import ru.sber.projectATM.common.handbook.Currency;
+
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Pattern;
+import java.sql.Array;
+import java.util.function.*;
+import java.util.*;
+
+@Slf4j
+@RestController
+
+public class controllerATM {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).